test/index.js

```diff
@@ -634,3 +634,16 @@ test('is.symbol', function (t) {
 
   t.end();
 });
+
+// no need to exhaustively test all negated methods. just need to test structure.
```

Comment:
Oops, thought I posted my reply.

Your tests test *your implementation*. The tests should test the same things whether or not it's done with reflection as you've done, or by manually adding negated methods for everything.

Testing `is.not.integer` proves `is.not.integer` works - testing `is.not.hex` does not - unless you want your tests to assume intimate knowledge of the implementation - but good tests have zero knowledge of the implementation beyond the public API and observable semantics.